scripts/gen_coverage_report.sh

```diff
@@ -26,10 +26,13 @@ function cov {
     cargo llvm-cov --no-report run --bin="$1" -- "${@:2}"
 }
 
+aria2c -x5 -d "$TMP_DIR" -o calibnet_snapshot.zst https://snapshots.calibrationnet.filops.net/minimal/latest.zst
 cargo llvm-cov --workspace clean
 cargo llvm-cov --workspace --no-report --features slow_tests
 cov forest-cli --chain calibnet db clean --force
 cov forest-cli --chain calibnet snapshot fetch --aria2 -s "$TMP_DIR"
+SNAPSHOT_PATH=$(find "$TMP_DIR" -name \*.zst | head -n 1)
+cov forest --chain calibnet --encrypt-keystore false --import-snapshot "$SNAPSHOT_PATH" --height=-200 --detach --track-peak-rss
 SNAPSHOT_PATH=$(find "$TMP_DIR" -name \*.car | head -n 1)
 cov forest --chain calibnet --encrypt-keystore false --import-snapshot "$SNAPSHOT_PATH" --height=-200 --detach --track-peak-rss
```

Comment:
This code tries to detach two instances of Forest. I don't think that will work (and the errors will be hidden in the output files). You probably just want to verify that Forest can load the compressed snapshot.

Reply:
You could even clean the DB between the two invocations.

Reply:
Good catch, fix. Thanks!